src/Drawer/Drawer.js

```diff
@@ -41,7 +41,8 @@ export const styleSheet = createStyleSheet('MuiDrawer', (theme) => {
       WebkitOverflowScrolling: 'touch', // Add iOS momentum scrolling.
     },
     docked: {
-      flex: '0 0 auto',
```

Comment:
Oh, its ok not include this change, which was a try to imitate old doc behavior.  But then i have no idea, how to include docked example in docs - since it cannot be included on some top level (as menu) and its somewhere at bottom in DOM,